docker/testing/ubuntu2004_j11.docker

```diff
@@ -95,17 +95,23 @@ ENV ANT_HOME=/usr/share/ant
 
 # run pip commands and setup virtualenv (note we do this after we switch to cassandra user so we
 # setup the virtualenv for the cassandra user and not the root user by accident) for Python 3.6/3.7/3.8
+# Don't build cython extensions when installing cassandra-driver. During test execution the driver
+# dependency is refreshed via pip install --upgrade, so that driver changes can be pulled in without
+# requiring the image to be rebuilt. Rebuilding compiled extensions is costly and is disabled by
+# default in test jobs using the CASS_DRIVER_X env vars below. However, if the extensions are
+# included in the base image, the compiled objects are not updated by pip at run time, which can
+# cause errors if the tests rely on new driver functionality or bug fixes.
 RUN virtualenv --python=python3.6 env3.6
 RUN chmod +x env3.6/bin/activate
-RUN /bin/bash -c "source ~/env3.6/bin/activate && pip3 install --upgrade pip && pip3 install Cython && pip3 install -r /opt/requirements.txt && pip3 freeze --user"
+RUN /bin/bash -c "export CASS_DRIVER_NO_CYTHON=1 CASS_DRIVER_NO_EXTENSIONS=1 && source ~/env3.6/bin/activate && pip3 install --upgrade pip && pip3 install -r /opt/requirements.txt && pip3 freeze --user"
```

Comment:
This is totally harmless, but `CASS_DRIVER_NO_EXTENSIONS` is all you need. It supersedes the more selective `CASS_DRIVER_NO_CYTHON`

Reply:
Yeah, I thought that was the case, but erred on the side of caution by following precedent - the scripts which run dtests in this repo & all the circle tasks set both, so I just followed them :)